dbms/src/Functions/GeoUtils.h

```diff
@@ -19,6 +19,11 @@
 #pragma GCC diagnostic pop
 #endif
 
+#if __clang__ && __clang_major__ <= 4
+#else
+#define USE_POINT_IN_POLYGON 1
```

Comment:
Why this is needed? The `pointInPolygon` is successfully compiled with gcc and already in the latest release candidate.